### DISCUSSION TITLE
Add examples using the Diviner, Prophet, and pmdarima libraries

### DISCUSSION THREAD

**Discussion by amotl** [crate-workbench/mlflow-cratedb#292] [Ideas]
## About

GH-13 adds an example program using the [Merlion] library. Thanks, @andnig.

On this matter, it may also be interesting to present a variant using [Diviner], see also [Diviner Documentation].
Currently, Diviner supports the [Prophet] and [pmdarima] open source libraries for forecasting at scale.

[Diviner]: https://github.com/databricks/diviner
[Diviner Documentation]: https://databricks-diviner.readthedocs.io/
[Merlion]: https://github.com/salesforce/Merlion
[pmdarima]: http://alkaline-ml.com/pmdarima/
[Prophet]: https://facebook.github.io/prophet/

**Comment by amotl** [comment]
I think this can be closed, considered to be superseded by GH-55. Thanks for contributing this, @andnig!

NB: Is PyCaret also wrapping those libraries mentioned above?

**Comment by andnig** [comment]
@amotl pycaret provides support for the prophet forecaster and pmdarima as optional arima forecasting backend. I'm not familiar with how diviner handles things - but maybe it's worth a post if you consider the wider databricks ecosystem relevant for Crate.

**Comment by amotl** [comment]
It looks like those are the corresponding canonical notebooks for Diviner+GroupedProphet/GroupedPmdarima. If those can easily be absorbed by PyCaret, i.e. if it provides the same features, there would be no need to specialize too much.

- [GroupedProphet example notebook](https://databricks-diviner.readthedocs.io/en/latest/tutorials-and-examples/notebooks/grouped_prophet_example_jupyter.html)
- [GroupedPmdarima example notebook](https://databricks-diviner.readthedocs.io/en/latest/tutorials-and-examples/notebooks/grouped_pmdarima_example_jupyter.html)
